apps/native/app/src/screens/document-detail/document-detail.tsx

```diff
@@ -265,15 +253,10 @@ export const DocumentDetailScreen: NavigationFunctionComponent<{
     if (hasPdf) {
       setLoaded(false)
     }
-    if (touched) {
```

Comment:
Snær removed this in his PR updating dependencies after talking to Enum so this is not necessary anymore.